web/admin/components/BusinessDropdown.js

```diff
@@ -0,0 +1,52 @@
+import MenuItem from "@mui/material/MenuItem";
+import Select from "@mui/material/Select";
+import React from "react";
+import { useUpdateEmployeeBusinessType } from "../../common/useUpdateBusiness";
+
+const BUSINESS_TYPES = [
+  {
+    id: "LONG_DISTANCE",
+    label: "Longue distance"
+  },
+  {
+    id: "SHORT_DISTANCE",
+    label: "Courte distance"
+  },
+  {
+    id: "SHIPPING",
+    label: "Messagerie, Fonds et valeur"
+  },
+  {
+    id: "FREQUENT",
+    label: "Lignes régulières"
+  },
+  {
+    id: "INFREQUENT",
+    label: "Occasionnels"
+  }
+];
+export function BusinessDropdown({ employmentId, companyId, business }) {
```

Comment:
- [ ] @sandrica89 Vérifier l'accessibilité du composant

Reply:
Checked with https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/
- Keyboar support : everything OK except space doesn't close the popup - but this is related to MUI component :white_check_mark: 
- Listbox role: OK :heavy_check_mark: 
- Listbox aria-expanded : OK :heavy_check_mark: 
- Listbox aria-haspopup : OK :heavy_check_mark: 
- Listbox aria-label ou aria-labelledby :  missing :negative_squared_cross_mark: 
- Option aria-selected: OK :heavy_check_mark: 

:orange_circle: Either add `aria-label="Type d'activité"` on the Select component, or `aria-labelledby`  with the `id` from the column name.